### PR TITLE
Update copy in empty dashboard services health panel

### PR DESCRIPTION
**Pull request by philipnrmn**
Before:

screen shot 2017-02-06 at 11 46 45 am

After:
screen shot 2017-02-06 at 11 46 16 am

### CODE REVIEW

**Comment by jfurrow** [comment]
:shipit:

File: plugins/services/src/js/components/ServiceList.js [hunk 1]
```diff
@@ -113,7 +112,7 @@ const ServiceList = React.createClass({
     return (
       <div>
         <h3 className="flush-top text-align-center">No Services Running</h3>
-        <p className="flush text-align-center">Use the {Config.productName} command line tools to find and install services.</p>
```
Comment (rcorral) [inline plugins/services/src/js/components/ServiceList.js]: I wonder if we should link to the Services page. What do you think?

Comment (philipnrmn) [inline plugins/services/src/js/components/ServiceList.js]: Makes sense :)

**Comment by philipnrmn** [comment]
Run integration tests

**Pull request merged**